xds/internal/client/client.go

```diff
@@ -401,9 +401,26 @@ type SecurityConfig struct {
 	RequireClientCert bool
 }
 
+// ClusterType is the type of cluster from a received CDS response.
+type ClusterType int
+
+const (
+	// Eds represents the EDS cluster type, which will delegate endpoint discovery
+	// to the management server.
+	Eds ClusterType = iota
```

Comment:
`EDS` should be capitalized https://github.com/golang/go/wiki/CodeReviewComments#initialisms

And rename those enum vars as `ClusterTypeEDS`, `ClusterTypeLogicalDNS`, `ClusterTypeAggregate`. Those are package level consts, names like `EDS` could cause conflicts later.

Reply:
Renamed.